### PR TITLE
📝 Ecommerce redirect method

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Add documentation for redirect method.
Sections include overview diagram and implementation instructions.

Test plan: Expect to section regarding redirect method on https://docs.centrapay.com/guides/ecommerce-website/

### CODE REVIEW

**Comment by MeganSteenkamp** [comment]
Demo site: http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/guides/ecommerce-website/

File: src/content/guides/ecommerce-website.mdx [hunk 1]
```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
...
+  Note over MP: Checkout complete ✅
+```
+
+### Implementation
+
+> Your merchant must be configured with allowed domains for your redirect URLs.
+
+1. Display Centrapay as a payment option on checkout.
+2. When the customer places an order using the Centrapay payment option, you are expected to [Create a Payment Request](https://docs.centrapay.com/api/payment-requests/#create-a-payment-request).
+  **You must define a `redirectPaidUrl` and a `redirectCancelUrl`.**
```
Comment (tom-blake) [inline src/content/guides/ecommerce-website.mdx:83]: @MeganSteenkamp Suggestion: Provide more explanation on what these URLs each mean

**Review by tom-blake — APPROVED** [review]

File: src/content/guides/ecommerce-website.mdx [hunk 2]
```diff
@@ -7,11 +7,13 @@ nav:
 ---
 import CodePanel from '../../components/CodePanel.astro';
 
-Centrapay enables businesses to process payments with connected Centrapay assets online. To process online payments, businesses need to integrate with one of our eCommerce payment flows.
+Centrapay enables businesses to process payments with connected Centrapay assets online. To process online payments, businesses need to integrate with one of our redirect or popup eCommerce payment flows.
 
 ## Know before you code
 
 Please get in touch with Centrapay to configure your business for eCommerce.
+If you decide to use the redirect flow, you will need to inform us of allowed domains for redirect URLs.
```
Comment (alexBinns) [inline src/content/guides/ecommerce-website.mdx:15]: Is it worth mentioning the merchant config here?

File: src/content/guides/ecommerce-website.mdx [hunk 3]
```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
...
+## Centrapay Checkout Sample
 
 A [sample Centrapay e-commerce application](https://github.com/centrapay/centrapay-checkout-sample) is available.
-It uses the Centrapay SDK on the frontend and Node.js on the backend.
+It includes demos of both the redirect method and the popup method.
 Please refer to the README.md file for configuration instructions.
 
+## Redirect Method
+
+The Redirect Method is the standard method used by most merchants.
```
Comment (alexBinns) [inline src/content/guides/ecommerce-website.mdx:36]: Do we mention that this is the safer method?

File: src/content/guides/ecommerce-website.mdx [hunk 4]
```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
...
+  CP-->>-MS: Return Payment Request
+  MS-->>-MP: Return Payment Request
+  end
+  MP-->>CP: Return
+  Note over MP: Checkout complete ✅
+```
+
+### Implementation
+
+> Your merchant must be configured with allowed domains for your redirect URLs.
```
Comment (alexBinns) [inline src/content/guides/ecommerce-website.mdx:79]: You have mentioned this twice

File: src/content/guides/ecommerce-website.mdx [hunk 5]
```diff
@@ -20,21 +22,86 @@ Centrapay will provide you with the following resources to allow you to start cr
...
+
+1. Display Centrapay as a payment option on checkout.
+2. When the customer places an order using the Centrapay payment option, you are expected to [Create a Payment Request](https://docs.centrapay.com/api/payment-requests/#create-a-payment-request).
+  **You must define a `redirectPaidUrl` and a `redirectCancelUrl`.**
+
+    Our payment protocol supports several optional extensions. Please review the extensions below and determine which ones you need for your integration.
```
Comment (alexBinns) [inline src/content/guides/ecommerce-website.mdx:85]: What about partial pay?

File: src/content/guides/ecommerce-website.mdx (hunk 3 above)
Comment (MeganSteenkamp) [reply src/content/guides/ecommerce-website.mdx:36]: @alexBinns safer how?

File: src/content/guides/ecommerce-website.mdx (hunk 4 above)
Comment (MeganSteenkamp) [reply src/content/guides/ecommerce-website.mdx:79]: @alexBinns yes I did because I thought people might not notice the first time 😂

**Pull request merged**